cyclonedx-bom/src/models/bom.rs

```diff
@@ -204,6 +213,46 @@ impl Bom {
         let bom: crate::specs::v1_4::bom::Bom = self.into();
         bom.write_xml_element(&mut event_writer)
     }
+
+    /// Parse the input as a JSON document conforming to [version 1.5 of the specification](https://cyclonedx.org/docs/1.5/json/)
+    pub fn parse_from_json_v1_5<R: std::io::Read>(
```

Comment:
I think we'll want to change the API from `parse_from_json_v1_5(R)` to `parse_json_with_version(version, R)` and make it accept the Version enum. But that doesn't have to happen as part of this PR. I'll open an issue to track this.